components/snackbar/Snackbar.js

```diff
@@ -39,15 +40,31 @@ const factory = (Overlay, Button) => {
     }
 
     componentWillReceiveProps (nextProps) {
+      const listenerTransition = () => {
+        this.setFocus();
+        this.refs.snackbar.removeEventListener('transitionend', listenerTransition);
+      };
+      if (nextProps.active) {
+        this.refs.snackbar.addEventListener('transitionend', listenerTransition);
```

Comment:
Instead, wouldn't it be better to add to `componentDidMount` and `componentWillUnmount`, register and unregister once, and simply handle it in the callback using props?.